guide/app_accuracy.tex

```diff
@@ -125,7 +125,7 @@ \section{Minor Bodies}
 
 Positions for the Minor Bodies (Dwarf Planets, Asteroids, Comets) are
 computed with standard algorithms found in astronomical text
-books. The generally used method of orbital elements allows to compute
+books. The generally used method of orbital elements allows the user to compute
```

Comment:
probably "allows computing". Not just the user of our program, but anybody who uses the scientific model.